public/js/app/components/CourseTitle.jsx

```diff
@@ -6,8 +6,10 @@ import { useFormatCredits } from '../hooks/useFormatCredits'
 
 const adminLink = (courseCode, languageShortname) => `/kursinfoadmin/kurser/kurs/${courseCode}?l=${languageShortname}`
 
-const CourseTitle = ({ courseTitleData = '', pageTitle }) => {
+const CourseTitle = ({ courseTitleData = '', pageTitle, preparatory }) => {
```

Comment:
Name the new prop courseLevelCode instead? Becuase `preparatory` is just one possible value?
```suggestion
const CourseTitle = ({ courseTitleData = '', pageTitle, courseLevelCode }) => {
```